data-prepper-plugins/aws-plugin-api/src/main/java/org/opensearch/dataprepper/aws/api/AwsCredentialsOptions.java

```diff
@@ -65,6 +74,10 @@ public Map<String, String> getStsHeaderOverrides() {
         return stsHeaderOverrides;
     }
 
+    public boolean isUseDefaultCredentials() {
```

Comment:
We already have a concept of `default` in the plugin configuration. So using "default" here is rather confusing.

https://github.com/opensearch-project/data-prepper/blob/81e4058652c44cddbf0eac123efb92473ea64fe0/data-prepper-plugins/aws-plugin/src/main/java/org/opensearch/dataprepper/plugins/aws/AwsPluginConfig.java#L12-L13

So I think we should use a different name here.

One idea: `isUseDefaultCredentialsProvider` to clarify that this is coupled with the SDK's `DefaultCredentialsProvider`.

Thoughts?

Reply:
Thanks @dlvenable for your input. I will make the change.